controllers/users.js

```diff
@@ -565,20 +576,45 @@ const markUnverified = async (req, res) => {
  * @param res {Object} - Express response object
  */
 
-const getUserImageForVerification = async (req, res) => {
+const getUserPhotoVerificationRequests = async (req, res) => {
   try {
-    const { id: userId } = req.params;
-    const userImageVerificationData = await userQuery.getUserImageForVerification(userId);
+    const { userId } = req.params;
+    const userData = req.userData;
+    if ((userData.id !== userId && !userData.roles[ROLES.SUPERUSER]) || !userId) {
+      return res.boom.unauthorized("You are not authorized to view this user's image verification data");
+    }
```

Comment:
Shouldn't middle ware be used for this?

Reply:
the case is-
- If the user is updating their photo-verification object they can
- photo-verification object of another user can only be updated, when the requester is a `SUPERUSER`